### PR TITLE
Fix typos in frontends and few obvious issues in onnx and pytorch

### PR DESCRIPTION
**Pull request by mvafin**
### Details:
 - *Fix typos in frontends*
 - *Fix obvious problems in ONNX and PyTorch frontends*

### CODE REVIEW

File: src/frontends/onnx/frontend/src/op/com.microsoft/qlinear_activation.cpp [hunk 1]
```diff
@@ -37,10 +39,11 @@ ov::OutputVector qlinear_activation(const ov::frontend::onnx::Node& node, const
     auto input_tensor = inputs[0];
     auto input_scale = inputs[1];
     auto input_zero_point =
-        (inputs[2].get_shape().empty()) ? v0::Constant::create(input_tensor.get_element_type(), {}, {0}) : inputs[2];
+        ov::op::util::is_null(inputs[2]) ? v0::Constant::create(input_tensor.get_element_type(), {}, {0}) : inputs[2];
     auto output_scale = inputs[3];
-    auto output_zero_point =
-        (inputs.size() > 4) ? inputs[4] : v0::Constant::create(input_tensor.get_element_type(), {}, {0});
+    auto output_zero_point = (inputs.size() > 4 && !ov::op::util::is_null(inputs[4]))
```
Comment (bumbosiepsak) [inline src/frontends/onnx/frontend/src/op/com.microsoft/qlinear_activation.cpp]: [nitpick] How about:
```
common::is_input_valid(4)
```
?

File: src/frontends/onnx/frontend/src/op/com.microsoft/quick_gelu.cpp [hunk 2]
```diff
@@ -34,8 +34,8 @@ ov::OutputVector quick_gelu(const ov::frontend::onnx::Node& node) {
                      "Unsupported input x type, accepted FP16, FP32, FP64, BFP16 but got: ",
                      element_type);
 
-    // Get attribute from node
-    const float alpha = node.get_attribute_value<float>("alpha");
+    // Get attribute from node; ORT default is 1.702
```
Comment (bumbosiepsak) [inline src/frontends/onnx/frontend/src/op/com.microsoft/quick_gelu.cpp]: [nitpick] Can be expressed with code (instead of comments):
```
constexpr auto ort_default = 1.702f;
const float alpha = node.get_attribute_value<float>("alpha", ort_default);
```

File: src/frontends/onnx/frontend/src/editor.cpp [hunk 3]
```diff
@@ -229,11 +229,17 @@ void graph_topological_sort(GraphProto* graph) {
...
         for (const auto& output : graph->output()) {
-            nodes_to_process.push(get_node_by_out_name(output.name()));
+            if (const auto* node = get_node_by_out_name(output.name())) {
+                nodes_to_process.push(node);
+            }
         }
 
         while (nodes_to_process.size() > 0) {
             auto* node = nodes_to_process.top();
+            if (node == nullptr) {
```
Comment (bumbosiepsak) [inline src/frontends/onnx/frontend/src/editor.cpp:239]: Question: will `node` ever be null? We're excluding such nodes while pushing to `nodes_to_process`.

File: tests/layer_tests/pytorch_tests/test_dict.py [hunk 4]
```diff
@@ -65,3 +65,25 @@ def test_dict_param_convert_model(self, ie_device, precision, ir_version):
...
+class aten_dict_mixed_inputs(torch.nn.Module):
+    def forward(self, x_dict: dict[str, torch.Tensor], y: torch.Tensor):
+        # one dict key is consumed alongside a regular tensor input; the other bypasses
+        return x_dict["x1"] + y, x_dict["x2"]
+
+
+class TestDictParamMixed(PytorchLayerTest):
+
+    def _prepare_input(self):
+        x1 = np.random.randn(1, 3, 4).astype(np.float32)
```
Comment (bumbosiepsak) [inline tests/layer_tests/pytorch_tests/test_dict.py]: Use static seed for generating input data.

File: src/frontends/onnx/frontend/src/op/com.microsoft/fusedgemm.cpp [hunk 5]
```diff
@@ -56,9 +57,12 @@ ov::OutputVector fusedgemm(const ov::frontend::onnx::Node& node) {
         double activation_alpha = node.get_attribute_value<double>("activation_alpha", 0.01);
         std::shared_ptr<ov::Node> activation_alpha_node =
             v0::Constant::create(input_c.get_element_type(), ov::Shape{1}, {activation_alpha});
-        return {std::make_shared<v0::PRelu>(gemm_res, activation_alpha_node)};
+        auto activation = std::make_shared<v0::PRelu>(gemm_res, activation_alpha_node);
+        return {activation->output(0)};
```
Comment (mvafin) [inline src/frontends/onnx/frontend/src/op/com.microsoft/fusedgemm.cpp]:
```suggestion
        return {std::make_shared<v0::PRelu>(gemm_res, activation_alpha_node)};
```

File: src/frontends/onnx/frontend/src/op/com.microsoft/fusedgemm.cpp [hunk 6]
```diff
@@ -56,9 +57,12 @@ ov::OutputVector fusedgemm(const ov::frontend::onnx::Node& node) {
...
     }
-    return {std::make_shared<v0::Relu>(gemm_res)};
+    CHECK_VALID_NODE(node, activation_type == "Relu", "Unsupported activation type for FusedGemm: ", activation_type);
+    auto activation = std::make_shared<v0::Relu>(gemm_res);
+    return {activation->output(0)};
```
Comment (mvafin) [inline src/frontends/onnx/frontend/src/op/com.microsoft/fusedgemm.cpp]:
```suggestion
    return {std::make_shared<v0::Relu>(gemm_res)};
```

File: src/frontends/onnx/frontend/src/op/com.microsoft/matmul_integer_to_float.cpp [hunk 7]
```diff
@@ -62,21 +63,26 @@ ov::OutputVector matmulintegertofloat(const ov::frontend::onnx::Node& node) {
 
     const auto matmul_result = std::make_shared<ov::op::v0::MatMul>(a_scaled, b_scaled);
 
-    if (inputs.size() > 6) {
+    if (inputs.size() > 6 && !ov::op::util::is_null(inputs[6])) {
```
Comment (mvafin) [inline src/frontends/onnx/frontend/src/op/com.microsoft/matmul_integer_to_float.cpp]:
```suggestion
    if (common::is_input_valid(node, 6)) {
```

File: src/frontends/onnx/frontend/src/op/com.microsoft/qlinear_activation.cpp (hunk 1 above)
Comment (mvafin) [inline src/frontends/onnx/frontend/src/op/com.microsoft/qlinear_activation.cpp]:
```suggestion
    auto output_zero_point = (common::is_input_valid(node, 4))
```

File: src/frontends/onnx/frontend/src/op/com.microsoft/qlinear_ops.cpp [hunk 8]
```diff
@@ -30,16 +32,18 @@ ov::OutputVector qlinear_op(const ov::frontend::onnx::Node& node, BinaryOp binar
...
     auto B = inputs[3];
     auto B_scale = inputs[4];
     auto B_zero_point =
-        (inputs[5].get_shape().empty()) ? v0::Constant::create(A.get_element_type(), {}, {0}) : inputs[5];
+        ov::op::util::is_null(inputs[5]) ? v0::Constant::create(B.get_element_type(), {}, {0}) : inputs[5];
 
     auto C_scale = inputs[6];
 
-    auto C_zero_point = inputs.size() > 7 ? inputs[7] : v0::Constant::create(C_scale.get_element_type(), {}, {0});
+    auto C_zero_point = (inputs.size() > 7 && !ov::op::util::is_null(inputs[7]))
```
Comment (mvafin) [inline src/frontends/onnx/frontend/src/op/com.microsoft/qlinear_ops.cpp]:
```suggestion
    auto C_zero_point = (common::is_input_valid(node, 7))
```

File: src/frontends/onnx/frontend/src/op/com.microsoft/fusedgemm.cpp [hunk 9]
```diff
@@ -6,6 +6,7 @@
 
 #include "core/null_node.hpp"
 #include "core/operator_set.hpp"
+#include "exceptions.hpp"
```
Comment (mvafin) [inline src/frontends/onnx/frontend/src/op/com.microsoft/fusedgemm.cpp]:
```suggestion
```

File: src/frontends/onnx/frontend/src/op/com.microsoft/fusedgemm.cpp [hunk 10]
```diff
@@ -58,6 +59,7 @@ ov::OutputVector fusedgemm(const ov::frontend::onnx::Node& node) {
             v0::Constant::create(input_c.get_element_type(), ov::Shape{1}, {activation_alpha});
         return {std::make_shared<v0::PRelu>(gemm_res, activation_alpha_node)};
     }
+    CHECK_VALID_NODE(node, activation_type == "Relu", "Unsupported activation type for FusedGemm: ", activation_type);
```
Comment (mvafin) [inline src/frontends/onnx/frontend/src/op/com.microsoft/fusedgemm.cpp]:
```suggestion
```

File: src/frontends/onnx/frontend/src/editor.cpp (hunk 3 above)
Comment (mvafin) [reply src/frontends/onnx/frontend/src/editor.cpp:239]: Unlikely, but doesn't hurt to check for null
